addons/packages/kpack/0.5.0/README.md

```diff
@@ -0,0 +1,565 @@
+
+   You can tail the logs the image that is currently building using the [kp cli](https://github.com/vmware-tanzu/kpack-cli/blob/main/docs/kp_build_logs.md)
+
+    ```
+    kp build logs tutorial-image -n default
+    ``` 
+
+   Once the image resource finishes building you can get the fully resolved built OCI image with `kubectl get`
+
+    ```
+    kubectl -n default get image tutorial-image
+    ```  
+
+   The output should look something like this:
+    ```
+    NAMESPACE   NAME                  LATESTIMAGE                                        READY
+    default     tutorial-image        index.docker.io/your-project/app@sha256:6744b...   True
+    ```
+
+   The latest OCI image is available to be used locally via `docker pull` and in a kubernetes deployment.
+
+8. Run the built app locally
+
+   Download the latest OCI image available in step #6 and run it with docker.
+
+   ```bash
+   docker run -p 8080:8080 <latest-image-with-digest>
+   ```
+
+   You should see the java app start up:
+   ```
+       
+              |\      _,,,--,,_
+             /,`.-'`'   ._  \-;;,_
+    _______ __|,4-  ) )_   .;.(__`'-'__     ___ __    _ ___ _______
+    |       | '---''(_/._)-'(_\_)   |   |   |   |  |  | |   |       |
+    |    _  |    ___|_     _|       |   |   |   |   |_| |   |       | __ _ _
+    |   |_| |   |___  |   | |       |   |   |   |       |   |       | \ \ \ \
+    |    ___|    ___| |   | |      _|   |___|   |  _    |   |      _|  \ \ \ \
+    |   |   |   |___  |   | |     |_|       |   | | |   |   |     |_    ) ) ) )
+    |___|   |_______| |___| |_______|_______|___|_|  |__|___|_______|  / / / /
+    ==================================================================/_/_/_/
+    
+    :: Built with Spring Boot :: 2.2.2.RELEASE
+   ``` 
+
+9. kpack rebuilds
+
+   We recommend updating the kpack image resource with a CI/CD tool when new commits are ready to be built.
+   > Note: You can also provide a branch or tag as the `spec.git.revision` and kpack will poll and rebuild on updates!
+
+   We can simulate an update from a CI/CD tool by updating the `spec.git.revision` on the image resource used in step #6.
+
+   If you are using your own application please push an updated commit and use the new commit sha. If you are using Spring Pet Clinic you can update the revision to: `4e1f87407d80cdb4a5a293de89d62034fdcbb847`.
+
+   Edit the image resource with:
+   ```
+   kubectl -n default edit image tutorial-image 
+   ``` 
+
+   You should see kpack schedule a new build by running:
+   ```
+   kubectl -n default get builds
+   ``` 
+   You should see a new build with
+
+   ```
+   NAME                                IMAGE                                          SUCCEEDED
+   tutorial-image-build-1-8mqkc       index.docker.io/your-name/app@sha256:6744b...   True
+   tutorial-image-build-2-xsf2l                                                       Unknown
+   ```
+
+   You can tail the logs for the image with the kp cli used in step #6.
```

Comment:
Global comment - need a consistent way to refer to steps when cross referencing. I think step 1, step 2 format is ok.